### PR TITLE
infinite HTML eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee-ing

### PR DESCRIPTION
**Pull request by Threebow**
We need HTML to be infinite like all the others

### CODE REVIEW

**Comment by J0sh0nat0r** [comment]
eee

**Comment by tilda** [comment]
EEEE :+1:

File: e.html [hunk 1]
```diff
@@ -2,11 +2,23 @@
 <html>
 
 <head>
-    <script>console.log('e');</script>
```
Comment (nightpool) [inline e.html:5]: Eee'e eeeee ee eee eeeeeee.eee eeeeeeeee

File: e.html [hunk 2]
```diff
@@ -2,11 +2,23 @@
...
 </head>
 
 <body>
-    <h1>e</h1>
+    <h1 e>e</h1>
+
+	<script>
+		let e = document.querySelector("[e]");
+		setInterval(() => {
+			e.innerHTML += "e";
```
Comment (RonjaPonja) [inline e.html:19]: eeeeeeeeeeeeeeeeeeeeeeeeeee eeeeeeeee eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee eeeeeeeee eeeeeeeee eeeeeeeeeeeeeeeeee!

**Review by RonjaPonja — APPROVED** [review]

**Comment by voidnull000** [comment]
>@HerrSpace approved these changes 32 minutes ago

Okay, this is epic.

File: e.html (hunk 2 above)
Comment (WeiChienHsu) [reply e.html:19]: eeeeeee!!! 👍 
eeeee, eeeeee? eeeeee....ee!!

**Review by andrewda — APPROVED** [review]
E 👍

**Comment by KishiTheMechanic** [comment]
Eee, eeeee.
Ee!? Ee. Eeee eee eeeeeeee.

- Eee ee

**Comment by LinuxMercedes** [comment]
EEEE EEe EE !!

**Pull request merged**